src/main/java/at/hannibal2/skyhanni/config/features/event/diana/DianaConfig.java

```diff
@@ -23,7 +23,8 @@ public class DianaConfig {
     public String color = "0:127:85:255:255";
 
     @Expose
-    @ConfigOption(name = "Guess Next Burrow", desc = "Uses math from §eSoopy's Guess Logic §7to find the next burrow. Does not require SoopyV2 or ChatTriggers to be installed.")
+    @ConfigOption(name = "Guess Next Burrow", desc = "Use math from §eSoopy's Guess Logic §7to find the next burrow.\n" +
+        "Does not require SoopyV2 or ChatTriggers to be installed.")
```

Comment:
yellow color feels right here. you only use the line break when the next line has a different color so far, and i like that consistency